### ISSUE TITLE
Private entries in shared calendars are visible to everyone

### ISSUE THREAD

**Issue by chaosjug** [owncloud/core#23983]
### Steps to reproduce
1. share a calendar
2. mark an entry as private (e.g. in Kontact or any other caldav client)
3. check if the entry is visible for others

### Expected behaviour
private entries should not be visible (and this was the pre 9.0 behavior)

### Actual behaviour
everyone who can see the calendar can see the private entries

### Server configuration
**Operating system:** Debian

**Web server:** nginx

**Database:** mysql

**PHP version:** 5

**ownCloud version:** 9.0.1

**Updated from an older ownCloud or fresh install:** updated

**Where did you install ownCloud from:** packages

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
No errors have been found.

**List of activated apps:**

Enabled:
  - activity: 2.2.1
  - calendar: 1.1
  - comments: 0.2
  - contacts: 1.1.0.0
  - dav: 0.1.6
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - tasks: 0.8.1
  - templateeditor: 0.1
  - updatenotification: 0.1.0
Disabled:
  - calendar__
  - contacts__
  - encryption  
  - external                                              
  - files_external                                                
  - user_external                                                  
  - user_ldap

**The content of config/config.php:**
{                                             
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",      
        "secret": "***REMOVED SENSITIVE VALUE***",            
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "\/var\/www\/owncloud\/data",  
        "overwrite.cli.url": "http:\/\/***REMOVED SENSITIVE VALUE***",       
        "dbtype": "mysql",                                
        "version": "9.0.1.3",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "php",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "loglevel": 2,
        "theme": "",
        "updatechecker": false
    }
}

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** firefox

**Operating system:** kubuntu

**Comment by DeepDiver1975** [comment]
Duplicate - we will fix this for 9.1

**Comment by chaosjug** [comment]
@DeepDiver1975 : Could you please tell which is the duplicate issue?
I wouldn't have opened a new one if I had found one for this problem...
